File: backend/IDE.Common/ModelsDTO/DTO/Project/ProjectUpdateDTO.cs
using IDE.Common.Enums;

namespace IDE.Common.DTO.Project
{
    public class ProjectUpdateDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int CountOfSaveBuilds { get; set; }
        public int CountOfBuildAttempts { get; set; }
        public AccessModifier AccessModifier { get; set; }
        public string Color { get; set; }
    }
}

File: frontend/src/app/models/DTO/Project/projectUpdateDTO.ts
import { AccessModifier } from "./../../Enums/accessModifier"

export interface ProjectUpdateDTO {
    id: number;
    name: string;
    description: string;
    countOfSaveBuilds: number;
    countOfBuildAttempts: number;
    accessModifier: AccessModifier;
    color: string;
}
